frontend/src/components/SecureExchange/AccessUsersPage.vue

```diff
@@ -1,5 +1,10 @@
 <template>
   <v-container>
+    <v-row>
+      <v-col :class="['d-sm-flex', 'align-center']">
+        <div>Primary EDX Code - <span id="primaryEdxActivationCode"> {{ this.primaryEdxActivationCode ? this.primaryEdxActivationCode.activationCode : 'code not found' }} </span></div>
+      </v-col>
+    </v-row>
```

Comment:
Please update the text to -   Code Not Found

Reply:
changed to capitals.